apps/fxc-front/src/app/components/2d/path-element.ts

```diff
@@ -254,6 +250,63 @@ export class PathElement extends connect(store)(LitElement) {
     this.postScoreToHost(score);
   }
 
+  private computeScore(points: LatLon[]): Score {
+    const track: ScoringTrack = {
+      points: points.map((point, i) => {
+        return {
```

Comment:
nit: you do not need to return if you add parents: `() => ({...})`

Reply:
ok. modified code